### PR TITLE
build: update cross angular deps and lock dependencies

### PR DESCRIPTION
**Pull request by devversion**
See individual commits. Follow-up to https://github.com/angular/angular/commit/249384b00c0f09a3e56757bb2d8db749d6d0ff6e

### CODE REVIEW

**Review by alan-agius4 — APPROVED** [review]

File: aio/package.json [hunk 1]
```diff
@@ -87,32 +87,32 @@
...
+    "@angular/service-worker": "15.0.0-next.5",
     "rxjs": "~7.5.0",
     "safevalues": "^0.3.1",
     "tslib": "^2.3.0",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
     "@angular-devkit/architect-cli": "0.1500.0-next.4",
-    "@angular-devkit/build-angular": "^14.2.1",
+    "@angular-devkit/build-angular": "15.0.0-next.4",
```
Comment (alan-agius4) [inline aio/package.json:109]: Nit: can update to `next.5` here?

Comment (devversion) [reply aio/package.json:109]: good catch. Let's actually leave that to Renovate to see if it properly functions now.

Comment (alan-agius4) [reply aio/package.json:109]: Can we just update this package and let renovate handle the other tooling packages? 

The reasoning behind it is that the update to 15.0.0-next.5 of build-angular is blocking one of my PRs (https://github.com/angular/angular/pull/47763)

Comment (alan-agius4) [reply aio/package.json:109]: Actually you can leave it as is, as we are not testing JIT mode. So I don't need the latest version.

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by jessicajaniuk — APPROVED** [review]
reviewed-for: size-tracking

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 4bddd90363c2e55bdca36832afb331e315640db4.

**Pull request closed without merging**